day2/index.js

```diff
@@ -0,0 +1,101 @@
+$(() => {
+  var skillPoint = 50;
+  var str = 0;
+  var dex = 0;
+  var int = 0;
+  var lux = 0;
+
```

Comment:
Use `let` instead of `var`.
`var` gives a different behaviour than what you understand in Java or other languages.
`let` will give you a `block scope` behaviour, which is what everyone preferred, just like in Java.

Further readings: [ความแตกต่างระหว่างการใช้ let กับ var](https://medium.com/pnpsolution/%E0%B8%84%E0%B8%A7%E0%B8%B2%E0%B8%A1%E0%B9%81%E0%B8%95%E0%B8%81%E0%B8%95%E0%B9%88%E0%B8%B2%E0%B8%87%E0%B8%A3%E0%B8%B0%E0%B8%AB%E0%B8%A7%E0%B9%88%E0%B8%B2%E0%B8%87%E0%B8%81%E0%B8%B2%E0%B8%A3%E0%B9%83%E0%B8%8A%E0%B9%89-let-%E0%B8%81%E0%B8%B1%E0%B8%9A-var-2b622a0732c1)

Reply:
Thank for suggestion.